jekyll/_cci2/configuration-reference.md

```diff
@@ -1674,7 +1674,17 @@ Truthiness rules are as follows:
 
 Logic statements always evaluate to a boolean value at the top level, and coerce as necessary. They can be nested in an arbitrary fashion, according to their argument specifications, and to a maximum depth of 100 levels.
 
-### Example
+### Logic Statement Examples
+
+```yaml
+workflows:
+  my-workflow:
+      when:
+        condition:
+          or:
+            - equal: [ master, << pipeline.git.branch >> ]
+            - equal: [ staging, << pipeline.git.branch >> ]
+```
```

Comment:
LGTM!

Could we maybe add subheaders?



#### Single Logic Statement:
```
      - when:
          condition:
            or:
              - equal: [ master, << pipeline.git.branch >> ]
              - equal: [ staging, << pipeline.git.branch >> ]
```

#### Nested Logic Statements:
```
    when:
      and:
        - not:
            equal: [ master, << pipeline.git.branch >> ]
        - or:
            - equal: [ canary, << pipeline.git.tag >> ]
            - << pipeline.parameters.deploy-canary >>
```